### ISSUE TITLE
Parse::Keyword dependency

### ISSUE THREAD

**Issue by jddurand** [tobyink/p5-kavorka#14]
Do you think it is possible to drop Parse::Keyword dependency ?
I refer to its DEPRECATED state in CPAN, and the its cpantesters matrix confirm this should be removed if possible.

**Comment by jmarca** [comment]
Ditto.  Trying to install Moops, which needs Kavorka, which needs Parse::Keyword, which refuses to install due to a test error and is no longer maintained.

Maybe I should look for an alternative to Moops?

**Comment by tobyink** [comment]
I don't think it would be easy. I would probably welcome any patches that do so though.

**Comment by tobyink** [comment]
New release of Parse::Keyword has a much better pass/fail matrix.

**Comment by jmarca** [comment]
Well, on a new install of Slackware 14.2, without any effort to tweak things by hand, the default CPAN install Parse::Keyword still fails to install.  And on the metacpan site the pod still says "deprecated do not use."  

I'm sure Parse::Keyword can be improved, as I was able to get it to compile with the Debian patch back in March, but as it stands now it isn't easy to just use cpan or whatever (cpanm) to install Moops - Kavorka - Parse::Keyword.

**Comment by jmarca** [comment]
Well, on a new install of Slackware 14.2 (current) (This is perl 5, version 24, subversion 0 (v5.24.0) built for x86_64-linux-thread-multi), without any effort to tweak things by hand, the default CPAN install Parse::Keyword still fails to install.  And on the metacpan site the pod still says "deprecated do not use."  

I'm sure Parse::Keyword can be improved, as I was able to get it to compile before with the Debian patch back in March, but as it stands now it isn't easy to just use cpan or whatever (cpanm) to install Moops - Kavorka - Parse::Keyword.

**Comment by srchulo** [comment]
Is it possible to use [Keyword::Simple](https://metacpan.org/pod/Keyword::Simple) instead? Or is it too limited?

**Comment by tobyink** [comment]
Keyword::Simple doesn't have things like `parse_listexpr`.
